CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+* v2.22.0
+- Adds the ability to track Core Web Vitals when the `trackCoreWebVitals` option is enabled. This also ensures that First Input Delay is reported from Firefox sessions.
```

Comment:
Was it just firefox or all non-chromium based browsers?

Reply:
It was for non-chromium based browsers, will update the comment now 👍